minimesos/src/main/java/com/containersol/minimesos/mesos/MesosAgent.java

```diff
@@ -56,16 +58,17 @@ public CreateContainerCmd getBaseCommand() {
         String hostDir = MesosCluster.getHostDir().getAbsolutePath();
 
         return dockerClient.createContainerCmd( getMesosImageName() + ":" + getMesosImageTag() )
-                .withName( getName() )
-                .withPrivileged(true)
-                .withEnv(createMesosLocalEnvironment())
-                .withPid("host")
-                .withLinks(new Link(getZooKeeperContainer().getContainerId(), "minimesos-zookeeper"))
-                .withBinds(
-                        Bind.parse("/var/run/docker.sock:/var/run/docker.sock"),
-                        Bind.parse("/sys/fs/cgroup:/sys/fs/cgroup"),
-                        Bind.parse(hostDir + ":" + hostDir)
-                );
+	    .withName( getName() )
+	    .withPrivileged(true)
+	    .withEnv(createMesosLocalEnvironment())
+	    .withPid("host")
+	    .withLinks(new Link(getZooKeeperContainer().getContainerId(), "minimesos-zookeeper"))
+	    .withBinds(
+		       Bind.parse("/var/run/docker.sock:/var/run/docker.sock"),
+		       Bind.parse("/sys/fs/cgroup:/sys/fs/cgroup"),
+		       Bind.parse(String.format("%s:%s:rw", hostDir + "/.minimesos/sandbox-" + getClusterId() + "/" + MesosCluster.SLAVE_MAPPING_DIR + this.getUuid(), MESOS_AGENT_SANDBOX_DIR)),
+		       Bind.parse(hostDir + ":" + hostDir)
+		       );
```

Comment:
There are 2 directories important in minimesos. The first is the _install directory_ at ${HOME}/.minimesos which contains `bin/minimesos` and the second is the _cluster directory_ at `${PWD}/.minimesos` which contains `minimesos.cluster`

In this snippet it seems you create a `sandbox-clusterID` directory in the install directory. Can't we instead add a `sandbox` directory in the cluster directory?

Let's also rename identifier like `hostDir` so it is clear which directory we are talking about.